src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ExtensionDataObject.cs

```diff
@@ -13,15 +13,15 @@ public sealed class ExtensionDataObject
     {
         private IList<ExtensionDataMember> _members;
 
-#if USE_REFEMIT
+#if USE_REFEMIT || NET_NATIVE
```

Comment:
@weshaggard I opened https://github.com/dotnet/corefx/issues/15892 for tracking both of these. Just to clarify, do we want to use "uapaot" or "UAPAOT"?